playwright-e2e/page-objects/local-authority/laBirthCertificateDetailsSection.page.ts

```diff
@@ -0,0 +1,107 @@
+import { Locator, Page } from '@playwright/test';
+
+export class LAChildSexAtBirthPage {
+  readonly childSexAtBirthHeading: Locator;
+
+  readonly maleRadioButton: Locator;
+  readonly femaleRadioButton: Locator;
+  readonly otherRadioButton: Locator;
+
+  readonly otherSexLabel: Locator;
+
+  readonly saveAndContinueButton: Locator;
+  readonly saveAsDraftButton: Locator;
+
+  constructor(page: Page) {
+    this.childSexAtBirthHeading = page.getByRole('heading', { name: "What was the child's sex at" });
+
+    this.maleRadioButton = page.getByLabel('Male', { exact: true });
+    this.femaleRadioButton = page.getByLabel('Female', { exact: true });
+    this.otherRadioButton = page.getByLabel('Other', { exact: true });
+
+    this.otherSexLabel = page.getByLabel("For example, if the child's");
+
+    this.saveAndContinueButton = page.getByRole('button', { name: 'Save and continue' });
+    this.saveAsDraftButton = page.getByRole('button', { name: 'Save as draft' });
+  }
+
+  async checkMaleRadioButton(): Promise<void> {
+    await this.maleRadioButton.check();
+  }
+
+  async checkFemaleRadioButton(): Promise<void> {
+    await this.femaleRadioButton.check();
+  }
+
+  async checkOtherRadioButton(): Promise<void> {
+    await this.otherRadioButton.check();
+  }
+
+  async fillOtherSexLabel(): Promise<void> {
+    await this.otherSexLabel.fill('intersex');
+  }
+
+  async clickSaveAndContinue(): Promise<void> {
+    await this.saveAndContinueButton.click();
+  }
+
+  async clickSaveAsDraft(): Promise<void> {
+    await this.saveAsDraftButton.click();
+  }
+}
+
+export class LAChildNationalityPage {
+  readonly childNationalityHeading: Locator;
+
+  readonly britishCheckbox: Locator;
+  readonly irishCheckbox: Locator;
+  readonly differentCountryCheckbox: Locator;
+  readonly notSureCheckbox: Locator;
+
+  readonly countryNameLabel: Locator;
+
+  readonly saveAndContinueButton: Locator;
```

Comment:
[since you are using save](https://github.com/hmcts/adoption-web/blob/dea8ca488600ab3b27e37314af2385fa3ec51968/playwright-e2e/pages/basePage.page.ts#L21)